### PR TITLE
Mat 4745/fixingTheExtractingSimilarMultipleLibraries

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-4745](https://jira.cms.gov/browse/MAT-4745)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/main/java/gov/cms/madie/madiefhirservice/services/LibraryService.java [hunk 1]
```diff
@@ -58,18 +58,11 @@ public boolean isLibraryResourcePresent(String name, String version) {
...
-    log.info("Getting Optional Library");
     if (optional.isPresent()) {
-      Library library=optional.get();
-      log.info("Name of Library",library.getName());
-      log.info("Library Version", library.getVersion());
-      log.info("Library Id",library.getId());
       return getCqlFromHapiLibrary(optional.get());
-
     } else {
-      log.info("throwing not found exception");
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/madiefhirservice/services/LibraryService.java]: Alternatively to deleting them, these log statements could be changed to either debug or trace level.

File: src/main/java/gov/cms/madie/madiefhirservice/services/LibraryService.java [hunk 2]
```diff
@@ -58,18 +58,11 @@ public boolean isLibraryResourcePresent(String name, String version) {
...
   private String processBundle(String name, String version, Bundle bundle) {
-    log.info("inside the process bundle");
+
     Optional<Library> optional = hapiFhirServer.findLibraryResourceInBundle(bundle, Library.class);
-    log.info("Getting Optional Library");
     if (optional.isPresent()) {
-      Library library=optional.get();
-      log.info("Name of Library",library.getName());
-      log.info("Library Version", library.getVersion());
-      log.info("Library Id",library.getId());
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/madie/madiefhirservice/services/LibraryService.java]: These 3 logging statements could be reduced to a single line with substitutions, though that's matter of preference of coding standards and debugging practices.

```log.debug("Library Name: {}, Version: {}, Id: {}", library.getName(), library.getVersion(), library.getId);```

**Review by jkotanchik-SB — APPROVED** [review]
